connect-file-pulse-api/src/main/java/io/streamthoughts/kafka/connect/filepulse/data/StructSchema.java

```diff
@@ -110,6 +110,14 @@ public List<TypedField> fields() {
         return ordered;
     }
 
+    public List<TypedField> fieldsByIndex() {
```

Comment:
Could you please just rewrite your method as follow to be consistent with `fields` method: 

```
    public List<TypedField> fieldsByIndex() {
        ArrayList<TypedField> ordered = new ArrayList<>(fields.values());
        ordered.sort(Comparator.comparing(TypedField::index));
        return ordered;
    }
```

Thanks :)

Reply:
In english, "ordered" usually implies: "ordered by natural order or alphabetical". In this case, it is ordered by the column index so I feel "byIndex" is more appropriate and it expresses more the intention. I will change it to "orderedByIndex". Consistency should be used only for cases truly consistent

Reply:
Done.